.github/workflows/main.yml

```diff
@@ -26,6 +26,24 @@ jobs:
       - name: Test Docker Image
         run: docker/build-local.sh
 
+  error-prone-checks:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Maven Install
+        run: |
```

Comment:
unrelated. i couldn't find information what shell it's being run in and whether it has `set -eu` enabled.
Did you?

Reply:
According to https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idsteps it runs with

```
bash --noprofile --norc -eo pipefail {0}
```